src/commands/scheduled/recently-published.ts

```diff
@@ -6,16 +6,73 @@ export default class ScheduledRecentlyPublished extends Command {
     "Describe our most recently published articles and podcast episodes"
 
   async run() {
+    const blog = await this.buildBlogSummary()
     const podcast = await this.buildPodcastSummary()
     const callToAction = this.buildCallToAction()
 
     const response = {
-      blocks: [...podcast, ...callToAction],
+      blocks: [...blog, ...podcast, ...callToAction],
     }
 
     this.log(JSON.stringify(response))
   }
 
+  async buildBlogSummary() {
+    const BLOG_FEED_URL = "https://artsy.github.io/feed.xml"
+
+    const parser = new Parser()
+    const feed = await parser.parseURL(BLOG_FEED_URL)
+
+    const mostRecentPost = feed.items[0]
+    const { pubDate } = mostRecentPost
+    const daysAgo = computeDaysAgo(pubDate)
+
+    const threeMostRecentPosts = feed.items.slice(0, 3)
+    const threeMostRecentPostBlocks = threeMostRecentPosts
+      .map(this.buildBlogArticleBlocks)
+      .flat()
+
+    const blocks = [
+      {
+        type: "section",
+        text: {
+          type: "mrkdwn",
+          text: `*${daysAgo} days ago* we published our most recent article on the *Artsy Engineering Blog*. Here are our most recent posts -- read and share them!`,
+        },
+      },
+      ...threeMostRecentPostBlocks,
+    ]
+
+    return blocks
+  }
+
+  buildBlogArticleBlocks(post: Parser.Item): Array<any> {
+    const { link, pubDate, title } = post
+    const formattedDate = formatDate(pubDate)
+    const blocks = [
+      {
+        type: "section",
+        text: {
+          type: "mrkdwn",
+          text: `<${link}|${title}>`,
+        },
+      },
+      {
+        type: "context",
+        elements: [
+          {
+            type: "mrkdwn",
+            text: formattedDate,
```

Comment:
@dblandin in #26 you suggested putting the author here; unfortunately, we don't expose the authors via the RSS feed. I'm going to follow up on this from the artsy.github.io side.